packages/analytics/analytics-chart/src/components/chart-plugins/ChartLegend.vue

```diff
@@ -143,7 +143,7 @@ const position = inject('legendPosition', ref(ChartLegendPosition.Right))
   }
 
   &::-webkit-scrollbar-track {
-    background-color: var(--white);
+    background-color: $kui-color-background;
     border-radius: 10px;
```

Comment:
Won't `kui-border-radius-40` (8px) work here?

Reply:
No, some of these border-radius properties are on purpose